src/client/pages/Erfpacht/DossierDetail/DataTableFacturen.tsx

```diff
@@ -11,30 +11,45 @@ import { ErfpachtDatalistProps } from './DatalistGeneral';
 import styles from './ErfpachtDossierDetail.module.scss';
 import classNames from 'classnames';
 
-export function DataTableFacturen({ dossier }: ErfpachtDatalistProps) {
+export function DataTableFacturen({
+  dossier,
+  relatieCode,
+}: ErfpachtDatalistProps) {
   const { displayPropsAlleFacturen, colStyles } = useErfpachtV2Data();
   const { dossierNummerUrlParam } = useParams<{
     dossierNummerUrlParam: string;
   }>();
+  const mailBody = `Dossiernummer: ${
+    dossier.dossierNummer
+  }%0D%0ARelatiecode: ${dossier.relaties?.find((relatie) => relatie.betaler)
+    ?.relatieCode}%0D%0ADebiteurnummer: ${dossier.facturen?.debiteurNummer}`;
+  const betaler = dossier.relaties?.find((relatie) => relatie.betaler);
+  const isBetaler = betaler?.relatieCode === relatieCode;
   const facturenBetalerDebiteurRows = [
     {
       rows: [
         {
           label: dossier.facturen.titelBetaler,
-          content: dossier.facturen.betaler,
+          content: dossier.facturen.betaler || '-',
```

Comment:
why '-'?

Reply:
Als indicatie dat er geen betaler is. Lijkt me beter dan geen waarden tonen

Reply:
Al is dit geen stricte requirement.